src/Streamly/Tutorial.hs

```diff
@@ -259,11 +259,14 @@ import Control.Monad.Trans.Class   (MonadTrans (lift))
 
 -- $flavors
 --
--- Streams can be combined using '<>' or 'mappend' to form a composite
--- stream. Deep streams are combined depth first i.e. each stream is traversed
--- fully before we traverse the next stream. Wide streams are combined breadth
--- first i.e. one element from each stream is traversed before coming back to
--- the first stream again.
+-- Composite streams can be interpreted in a depth first or breadth
```

Comment:
Looks good. Just one point, we have not hinted what is a composite stream or how it can be created e.g. using `<>` or `mappend`, that gives a more concrete idea about what we are talking about here.

Reply:
Good point!  I've added the first sentence back in